cirrus/server/cirrus/main.py

```diff
@@ -149,15 +153,23 @@ class EnrollmentMetricData(NamedTuple):
 
 
 def collate_enrollment_metric_data(
-    enrolled_partial_configuration: dict[str, Any]
+    enrolled_partial_configuration: dict[str, Any], nimbus_preview_flag: bool
 ) -> list[EnrollmentMetricData]:
     events: list[dict[str, Any]] = enrolled_partial_configuration.get("events", [])
     data: list[EnrollmentMetricData] = []
     for event in events:
         if event.get("change") == "Enrollment":
             experiment_slug = event.get("experiment_slug", "")
             branch_slug = event.get("branch_slug", "")
-            experiment_type = app.state.remote_setting.get_recipe_type(experiment_slug)
+            experiment_type = None
+            if nimbus_preview_flag:
+                experiment_type = app.state.remote_setting.get_preview_recipe_type(
+                    experiment_slug
+                )
+            else:
+                experiment_type = app.state.remote_setting.get_live_recipe_type(
+                    experiment_slug
+                )
```

Comment:
And then this stays the same just calls into two different remote settings instances instead of the same one with two methods.